profiles/lib/way_handlers.lua

```diff
@@ -521,7 +521,7 @@ function WayHandlers.blocked_ways(profile,way,result,data)
     local construction = way:get_value_by_key('construction')
 
     -- Of course there are negative tags to handle, too
-    if construction and construction ~= 'no' and construction ~= 'widening' then
+    if construction and construction ~= 'no' and construction ~= 'widening' and construction ~= 'minor' then
```

Comment:
Capturing from chat: I was under the misconception that this tagging scheme would try to parse `highway=construction and construction=highway_type`. I am fine with this change for the special cases to handle deprecated tags.